docs/user-guides/workspace-access/windsurf.md

```diff
@@ -0,0 +1,49 @@
+# Windsurf
+
+[Windsurf](https://codeium.com/windsurf) is Codeium's code editor designed for AI-assisted
+development.
+
+Follow this guide to use Windsurf to access your Coder workspaces.
+
+If your team uses Windsurf regularly, ask your Coder administrator to add Windsurf as a workspace application in your template.
+
+## Install Windsurf
+
+Windsurf can connect to your Coder workspaces via SSH:
+
+1. [Install Windsurf](https://docs.codeium.com/windsurf/getting-started) on your local machine.
+
+1. Open Windsurf and select **Get started**.
+
+   Import your settings from another IDE, or select **Start fresh**.
+
+1. Complete the setup flow and log in or [create a Codeium account](https://codeium.com/windsurf/signup)
+   if you don't have one already.
+
+## Install the Coder extension
```

Comment:
We need to replace this section with our OpenVSX listing. No need for manually installing the Coder extension now.
https://open-vsx.org/extension/coder/coder-remote